ccnmtldjango/template/+package+/settings_production.py_tmpl

```diff
@@ -3,13 +3,13 @@ from settings_shared import *
 import os
 
 TEMPLATE_DIRS = (
-    os.path.join(os.path.dirname(__file__), "templates"),
```

Comment:
Good point. Yeah, I think we can take advantage of this to shrink staging/prod configs a bit.